source/project-stories/contiki-holidays.erb

```diff
@@ -0,0 +1,399 @@
+  image: "/project-stories/contiki/team-meeting.jpg"
+  image_description: "Team meeting"
+
+statistics_1:
+  title: "Contiki in numbers"
+  number: "78%"
+  text: "the increased likelihood of users willing to book online"
+
+statistics_2:
+  number: "43%"
+  text: "conversion rate increase from previous period"
+
+statistics_3:
+  number: "47%"
+  text: "increase in bookings over previous period"
+
+agile:
+  title: "Agile key in move to iterative customer-led design development"
+  content: "Unboxed’s offshore Cape Town team, led Contiki through the Agile-based project – focusing on rapid delivery of customer features over ‘pixel perfect’ designs. The complete project team consisted of four developers, a Scrum Master, Contiki’s design team and the Product Owner and the total project duration was 16 weeks."
+  image: "/project-stories/contiki/design-presentation.jpg"
+  image_description: "Presentation of a design"
+
+project_quote:
+  quote: "Our ambitious business objectives around performance and revenue were overwhelmingly met. The project came in on time and budget. Can’t ask for more than that!"
+  source: "Simon Larkin, Product Owner, Contiki"
+
+collaboration:
+  title: "A step change in site performance shifts dated business model"
+  content: "The development and design teams worked side-by-side and communicated using ‘The Unboxed Way’ - frequent, high-bandwidth, clear and open channels. Contiki were able to re-prioritise their features on a weekly basis as new requirements emerged. Built in Ruby on Rails, with JavaScript elements and Unboxed’s own micro-service APIs, this is a site created to meet Contiki’s future business goals."
+  image: "/project-stories/contiki/sketch.jpg"
+  image_description: "Sketch of a design"
+
+release_quote:
+  quote: "We had a straightforward and smooth product release, with minimal fuss or user support issues."
+  source: "Simon Larkin, Contiki"
+
+objectives:
+  title: "Pre-project objectives met – results dropped straight onto bottom line"
+  content: "All project’s objectives have been met or exceeded. Page loading times have decreased by 56%, conversion rates are up 43% and bookings up 47%. The technologies employed are designed to dramatically cut the cost of future improvements."
+  image: "/project-stories/contiki/blue-screen.jpg"
+  image_description: "Contiki website screenshot"
+
+team_quote:
+  quote: "The 'Unboxed Way' created a fully integrated team and made offshoring development a reality for Contiki."
+  source: "Simon Larkin, Contiki"
+
+team:
+  people_active:
+    - short_name: "carlo"
+      full_name: "Carlo Kruger"
+      link: "/people/carlo-kruger"
+    - short_name: "patrick"
+      full_name: "Patrick Vine"
+      link: "/people/patrick-vine"
+    - short_name: "richard"
+      full_name: "Richard Archer"
+      link: "/people/richard-archer"
+    - short_name: "heidi"
+      full_name: "Heidi Geisler"
+      link: "people/heidi-geisler"
+  people_archived:
```

Comment:
I think it might be best to handle these archived people on the people page, rather than a per project basis. It saves us having to update all the current project stories if somebody leaves, instead leaving this responsibility to be done elsewhere (on the people page itself). So perhaps we could have this list including Ben and Gavin with them linking to their own people page (despite it being a 404 - it's a currently problem which we will be addressing in the future).